CMSgov/mint-app
# [EASI-3323] - Send Feedback Form

# EASI-3323

## Changes and Description

- Send feedback form, mutation, error state
- Style tweaks to footer
- Unit tests




## How to test this change

- Login as MINT or add your user to LD for flag `feedbackEnabled`
- Click the footer link to send feedback
- Verify the default first radio is set to No
- Submit the form as is/nothing should be required
- Verify submission page and link
- Verify closing the tab works as expected

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
